composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "raulfraile/ladybug": "~1.0.8",
-        "phpspec/prophecy": "~1.3.1"
+        "mikey179/vfsStream": "~1"
```

Comment:
Following the manual of that package :) no problem with using a more specific version.